### PR TITLE
add issues templates

### PR DESCRIPTION
**Pull request by CerineBnsd**
# Summary
This PR adds different issue templates. Templates help organizing the issues in categories, and help making sure that all the needed details are provided. (See an [example of issue template chooser](https://github.com/RITlug/teleirc/issues/new/choose)) 

# Description
The following templates are added:
- **New feature:** Suggest a new idea for the project.
- **Improvement:** Suggest an improvement to an existing functionality.
- **Refactor:** Refactor the codebase, or a part of it.
- **Bug Report:** Report a bug in the project.
- **Testing:** Add, improve or edit tests in the codebase.

The `config.yml` file is used to customize the templates chooser. (see: [Configuring the template chooser](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser))

# Further work
Once the custom labels are configured for the organization, we can add labels (metadata) in the templates. This will automatically tag the created issue with the specified labels.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# .github
\ No newline at end of file
+# GDG Algiers contributing guidelines
```
Comment (youben11) [inline README.md]: Shouldn't it include something like "default community health files"?

File: .github/ISSUE_TEMPLATE/new-feature.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Proposed solution
+Describe in details how you think the feature should be implemented.
+
+## Outcome 
+Describe briefly what the end results will be, once this issue is solved. 
+
+# Tests
+## Affected features
+State the different existing features that will be broken/affected by this new feature.
```
Comment (youben11) [inline .github/ISSUE_TEMPLATE/new-feature.md]: what about "borken/affected (if any)"?

File: .github/ISSUE_TEMPLATE/improvement.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+# Background
+## Motivation and context
+Explain what make you think this enhancement is needed. Make sure to link any exisiting Issues (or Pull Requests) that are related to this one (Whether they are closed or not).
+
+## Suggested Solution
+Describe clearly the solution you'd like to have, and any other possible alternative. 
+
+## Outcome
+Describe briefly what the end results will be, once this issue is solved.
```
Comment (youben11) [inline .github/ISSUE_TEMPLATE/improvement.md]: typo: "end result"

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+## Demonstration
+Add anything that can help better understand what's happening. For example: Screenshots and error Logs. 
+
+### Screenshots
+<!--screeshots here-->
+
+### Logs
+<!--logs here-->
+```
+Past logs here between the quotes
```
Comment (youben11) [inline .github/ISSUE_TEMPLATE/bug-report.md]: I think they are called "backticks". But we can just keep the "Past logs here"

**Review by youben11 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CerineBnsd) [reply README.md]: Exactly, I wanted to discuss this, and yes, this is more expressive.

**Review by youben11 — APPROVED** [review]

**Pull request merged**